TechnologyAssessmentApp/TechnologyAssessmentApp/Models/ArticleViewModel.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  ArticleViewModel.swift
+//  TechnologyAssessmentApp
+//
+//  Created by Kshitij Godara on 24/08/18.
+//  Copyright © 2018 Nagarro. All rights reserved.
+//
+
+import Foundation
+
+struct ArticleViewModel: Codable {
+    let status, copyright: String
+    let numResults: Int
+    let results: [Result]
+    enum CodingKeys: String, CodingKey {
+        case status, copyright
+        case numResults = "num_results"
+        case results
+    }
+}
+
+struct Result: Codable {
```

Comment:
why did you named this `Result` and not `Article` ?
Ask yourself what does this struct does ? From my understand, it represent a struct which is an article.

Reply:
Corrected accordingly